### ISSUE TITLE
Dragon paths

### ISSUE THREAD

**Issue by LockeExile** [zeldadungeon/maps#152]
Add a toggle to show dragon paths. Should go in a new menu along with toggles for region borders, etc.

**Comment by ARandomGitHubUser1** [comment]
I was going to add wandering merchants along with their paths; should I continue doing them as a pin type or should it become a part of this?

**Comment by LockeExile** [comment]
Yes I'd prefer those not be a pin type. You could add it to the Settings menu for now or make the new menu - I'm thinking an eye icon, title something like "Show/Hide Features", maybe move some of the existing Settings there.

**Comment by ARandomGitHubUser1** [comment]
To be honest, I have no clue how I'd go about doing any of that. I'll create a PR making them a pin type as a starting point, though.
